industrial_ci/rosdep-install.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -x
+#!/bin/bash -ex
```

Comment:
If I were adding this change, I would add a comment why we need this because I can foresee I will easily forget why.
But having `-x` isn't common already, so it might be obvious enough.